policyengine_us/parameters/gov/local/ca/riv/general_relief/income/deductions/sources.yaml

```diff
@@ -0,0 +1,13 @@
+description: Riverside county deducts these mandatory wage deductions from earned income under the General Relief Program.
+values:
+  2020-01-01:
+    - state_income_tax_reported
```

Comment:
I wonder if withheld income taxes are closer to reality here than actual computed tax amount:
`state_withheld_income_tax`